ucrop/build.gradle

```diff
@@ -9,6 +9,7 @@ repositories {
 }
 
 android {
+    namespace "com.yalantis.ucrop"
```

Comment:
Minor (🔍): Now that this is defined here the whole [AndroidManifest.xml](https://github.com/Automattic/uCrop/blob/develop-non-native/ucrop/src/main/AndroidManifest.xml) file could be deleted from within the `ucrop` module.

Reply:
Yea, no, bye-bye XML and thanks for all the fish (pain)! 🤪